includes/gateways/paypal/admin/notices.php

```diff
@@ -0,0 +1,59 @@
+<?php
+/**
+ * PayPal Admin Notices
+ *
+ * @package   easy-digital-downloads
+ * @copyright Copyright (c) 2021, Sandhills Development, LLC
+ * @license   GPL2+
+ * @since     2.11
+ */
+
+namespace EDD\PayPal\Admin;
+
+add_action( 'admin_notices', function () {
+	if ( ! current_user_can( 'manage_options' ) ) {
+		return;
+	}
+
+	// Bail if this notice has already been dismissed.
+	if ( get_user_meta( get_current_user_id(), '_edd_paypal_commerce_dismissed' ) ) {
+		return;
+	}
+
+	$enabled_gateways = array_keys( edd_get_enabled_payment_gateways() );
+	$enabled_gateways = array_diff( $enabled_gateways, array( 'paypal_commerce' ) );
+
+	// Show a notice if any PayPal gateway is enabled, other than PayPal Commerce.
+	$paypal_gateways = array_filter( $enabled_gateways, function( $gateway ) {
+		return false !== strpos( strtolower( $gateway ), 'paypal' );
+	} );
+
+	if ( ! $paypal_gateways ) {
+		return;
+	}
+
+	$dismiss_url = add_query_arg( array( 'edd_action' => 'dismiss_notices', 'edd_notice' => 'paypal_commerce' ) );
+	$setup_url = add_query_arg( array(
+		'post_type' => 'download',
+		'page'      => 'edd-settings',
+		'tab'       => 'gateways',
+		'section'   => 'paypal_commerce'
+	), admin_url( 'edit.php' ) );
+
+	?>
+	<div class="notice notice-info is-dismissible">
+		<h2><?php esc_html_e( 'Enable the new PayPal gateway for Easy Digital Downloads' ); ?></h2>
+		<p>
+			<?php
+			printf(
+				/* Translators: %1$s opening anchor tag; %2$s closing anchor tag */
+				esc_html__( 'A new, improved PayPal experience is now available in Easy Digital Downloads. %1$sClick here to activate it.%2$s', 'easy-digital-downloads' ),
```

Comment:
What would you think of removing the "Click here to activate it" sentence and instead making this link a primary button, and the dismiss link a secondary button, and putting them side by side in the notice? Something like:

![image](https://user-images.githubusercontent.com/4853066/118703474-210f2800-b7e4-11eb-8c1c-10e05c8b0cd6.png)

(I know you said this language is still pending but I wanted to suggest it.)

Reply:
Love it - thank you!